aiida/cmdline/commands/cmd_node.py

```diff
@@ -162,6 +162,36 @@ def show(nodes, print_groups):
                     echo.echo(gr_specs)
 
 
+@verdi_node.command('attributes')
+@arguments.NODES()
+@options.DICT_KEYS()
+@options.DICT_FORMAT()
+@with_dbenv()
+def attributes(nodes, keys, fmt):
+    """Show the attributes of one or more nodes."""
+    for node in nodes:
+        echo.echo("Pk: {}".format(node.pk), bold=True)
+        attr_dict = node.attributes
+        if keys is not None:
+            attr_dict = {k: v for k, v in attr_dict.items() if k in keys}
+        echo.echo_dictionary(attr_dict, fmt=fmt)
+
+
+@verdi_node.command('extras')
+@arguments.NODES()
+@options.DICT_KEYS()
+@options.DICT_FORMAT()
+@with_dbenv()
```

Comment:
Thanks a lot!
Could you add a `--raw` option (we should have it already) that instead of printing `Pk: XXX` before every line just creates a dictionary `{"PK1": load_node(PK1).attributes, "PK2": load_node(PK2).attributes, ...)` and prints only that, so you can pipe it into another code able to read JSON or YAML? (Now one would have to grep away `Pk:`)

Same for `attributes`

Reply:
Do I understand correct that for `yaml` the format of the actual extras or attributes content is still printed in `json`? If so, that would be a bit unexpected, no?

Reply:
Ah ok, good to know, thanks